### PR TITLE
chore: resolve swiftlint errors and warnings for Analytics plugin

### PR DESCRIPTION
**Pull request by phantumcode**
## Description
- resolve swiftlint errors and warnings for Analytics plugin
- only contains formatting changes, no functional changes

## General Checklist

- [x] Added new tests to cover change, if needed
- [x] Build succeeds with all target using Swift Package Manager
- [x] All unit tests pass
- [x] All integration tests pass
- [x] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [x] Documentation update for the change if required
- [x] PR title conforms to conventional commit style
- [x] New or updated tests include `Given When Then` inline code documentation and are named accordingly `testThing_condition_expectation()`
- [x] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Analytics/PinpointEvent.swift [hunk 1]
```diff
@@ -117,12 +117,12 @@ extension PinpointEvent: CustomStringConvertible {
         """
         return string
```
Comment (thisisabhash) [inline AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Analytics/PinpointEvent.swift:118]: looks like theme was copied from Xcode as well with the text

File: AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Support/Utils/PinpointRequestsRegistry.swift [hunk 2]
```diff
@@ -29,11 +29,11 @@ import AWSPluginsCore
         )
     }
 
-    fileprivate func sources(for api: API) -> Set<AWSPinpointSource> {
+    func sources(for api: API) -> Set<AWSPinpointSource> {
```
Comment (thisisabhash) [inline AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Support/Utils/PinpointRequestsRegistry.swift:32]: is this change intended?

Comment (phantumcode) [reply AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Support/Utils/PinpointRequestsRegistry.swift:32]: yes, it's being used/accessed by `CustomPinpointHttpClientEngine` which is defined in the same file.  Swiftlint recommended changing it from `fileprivate` to `private` but that would break `CustomPinpointHttpClientEngine`.

**Review by thisisabhash — APPROVED** [review]
